src/components/ConnectorTile.tsx

```diff
@@ -0,0 +1,318 @@
+                                                    authenticatedRoutes.captures
+                                                        .create.params
+                                                        .connectorID,
+                                                    row.connector_tags[0].id
+                                                )
+                                            );
+                                        } else if (
+                                            row.connector_tags[0].protocol ===
+                                            'materialization'
+                                        ) {
+                                            navigate(
+                                                getPathWithParam(
+                                                    authenticatedRoutes
+                                                        .materializations.create
+                                                        .fullPath,
+                                                    authenticatedRoutes
+                                                        .materializations.create
+                                                        .params.connectorId,
+                                                    row.connector_tags[0].id
+                                                )
+                                            );
+                                        }
+                                    }}
+                                >
+                                    {row.connector_tags[0].protocol ===
+                                    'capture' ? (
+                                        <FormattedMessage id="connectorTable.actionsCta.capture" />
+                                    ) : (
+                                        <FormattedMessage id="connectorTable.actionsCta.materialization" />
+                                    )}
+                                </Button>
+                            </Stack>
+                        </Paper>
+                    </Grid>
+                ))
+            ) : (
+                <Grid item sx={{ width: '100%' }}>
+                    {isValidating ||
+                    tableState.status === TableStatuses.LOADING ? (
+                        <Box>
+                            <LinearProgress />
+                        </Box>
+                    ) : (
+                        <Paper
+                            elevation={0}
+                            sx={{
+                                height: '100%',
+                                borderRadius: 5,
+                                background: (theme) =>
+                                    theme.palette.mode === 'dark'
+                                        ? 'linear-gradient(160deg, rgba(172, 199, 220, 0.18) 2%, rgba(172, 199, 220, 0.12) 40%)'
+                                        : slate[50],
```

Comment:
This looks repeated. Can we move this to a shared spot?

Reply:
Yes, I plan to do so before opening the PR.